packages/connect-react/src/components/passkeyList/PasskeyListScreen.tsx

```diff
@@ -0,0 +1,188 @@
+import type { CorbadoConnectPasskeyListConfig } from '@corbado/types';
+import type { Passkey } from '@corbado/web-core/dist/api/v2';
+import log from 'loglevel';
+import React, { useCallback, useEffect, useState } from 'react';
+
+import useManageProcess from '../../hooks/useManageProcess';
+import useModal from '../../hooks/useModal';
+import useShared from '../../hooks/useShared';
+import { ManageScreenType } from '../../types/screenTypes';
+import { CorbadoTokens } from '../../types/tokens';
+import { Button } from '../shared/Button';
+import PasskeyList from './PasskeyList';
+
+const PasskeyListScreen = () => {
+  const { navigateToScreen, config } = useManageProcess();
+  const { setPasskeyListToken, passkeyListToken } = useManageProcess();
+  const { show, hide } = useModal();
+  const { getConnectService } = useShared();
+
+  const [passkeyList, setPasskeyList] = useState<Passkey[]>([]);
+  const [loading, setLoading] = useState<boolean>(true);
+  const [ac, _] = useState(new AbortController());
```

Comment:
Sharing the abortController can introduce some problems. 
Right now, the init call is cancelled twice, when React runs in development mode => passkey-list can not load (this explains why the playground does not work locally)